pkg/execute/kubectl_cmd_builder.go

```diff
@@ -280,27 +281,40 @@ func (e *KubectlCmdBuilder) tryToGetResourceNamesSelect(botName string, bindings
 
 	out, err := e.kcExecutor.Execute(bindings, cmd, true)
 	if err != nil {
-		return EmptyResourceNameDropdown(botName)
+		e.log.WithField("error", err.Error()).Error("Cannot fetch resource names. Returning empty resource name dropdown.")
+		return EmptyResourceNameDropdown()
 	}
 
 	lines := getNonEmptyLines(out)
 	if len(lines) == 0 {
-		return EmptyResourceNameDropdown(botName)
+		return EmptyResourceNameDropdown()
 	}
 
 	return ResourceNamesSelect(botName, overflowSentence(lines), state.resourceName)
 }
 
 func (e *KubectlCmdBuilder) tryToGetNamespaceSelect(ctx context.Context, botName string, bindings []string, details stateDetails) *interactive.Select {
-	resourceDetails := e.commandGuard.GetResourceDetails(details.verb, details.resourceType)
+	log := e.log.WithFields(logrus.Fields{
+		"state":    details,
+		"bindings": bindings,
+	})
+
+	resourceDetails, err := e.commandGuard.GetResourceDetails(details.verb, details.resourceType)
+	if err != nil {
+		log.WithField("error", err.Error()).Error("Cannot fetch resource details, ignoring namespace dropdown...")
```

Comment:
Is that proper behavior? Shouldn't we print the "internal error" message? 🤔 Looking at the guard logic once again, we can have `ErrResourceNotFound` error here which is rather impossible, and other error from K8s API. 🤔

EDIT: On the other hand even if we won't print the selectbox, the command for namespaced resource will be still valid, but executed in the  default namespace for a given channel, so...

Alright, let's keep it 👍

Reply:
Yep, I try to print as much as I can without printing the actual error, so great that I can keep it 👍